1-exercises/D-object-methods/exercise.js

```diff
@@ -8,10 +8,13 @@
 */
 
 let student = {
-  // write code here
-}
+  name: "Daniel",
```

Comment:
Please think about the object's methods as functions because basically, they are functions. 
You need to finish this exercise in a dynamic way.

Please check this: https://syllabus.codeyourfuture.io/js-core-1/week-1/lesson#functions